src/monitoring/monitoring.module.ts

```diff
@@ -30,6 +34,9 @@ import { SystemStateSnapshotRepository } from './system-state-snapshot.repositor
     StakingBalanceObserver,
     UserObserver,
     BankingBotObserver,
+    OlkypayObserver,
+    OlkypayService,
+    BankTxBatchRepository,
```

Comment:
Remove `OlkypayService` and `BankTxBatchRepository`. Export `OlkypayService` in `PaymentModule`.